cmd/podman/rmi.go

```diff
@@ -63,13 +63,14 @@ func rmiCmd(c *cli.Context) error {
 	for _, arg := range imagesToDelete {
 		image, err := runtime.GetImage(arg)
 		if err != nil {
-			return errors.Wrapf(err, "could not get image %q", arg)
+			fmt.Printf("could not get image %q: %v\n", arg, err)
 		}
 		id, err := runtime.RemoveImage(image, c.Bool("force"))
 		if err != nil {
-			return errors.Wrapf(err, "error removing image %q", id)
+			fmt.Printf("%v\n", err)
```

Comment:
Needs the same handling we have in a few other commands where we have a lastErr variable and assign to it if err != nil - as written, this code will let rmi exit cleanly if some images failed to remove

Reply:
fixed